loops-ramon-kendra.js

```diff
@@ -0,0 +1,61 @@
+// Logging values with for loops
+// Write a for loop that logs each number from 1 - 20.
+// for(let i=0; i<21; i++){
+//   console.log(i)
+// }
+// Write a for loop that logs the result of each number from 1 - 20 tripled.
+// for(let m=1; m<21; m++){
+//   console.log(m*3)
+// }
+
+// Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+// for (let i=1; i<21; i++){
+//   if(i%2 !==0){
+//     console.log("ODD")
+//   }else{
+//     console.log(i)
+//   }
+// }
+// Looping over an array. Consider this variable:
+// Create a loop that will log the highest number from the array. Expected output --> 67
+// let nums = [3, 57, -9, 20, 67]
+// let highest = nums[0]
+// for(let i=0; i<nums.length; i++){
+//   if(nums[i]>highest){
+//       highest = nums[i]
+//   }
+// }
+// console.log(highest)
+
+
+// Create a loop that will log the lowest number from the array Expected output --> -9
+
+// let nums = [3, 57, -9, 20, 67]
+// let lowest = nums[0]
+// for(let i=0; i<nums.length; i++){
+//   if(nums[i]<lowest){
+//       lowest = nums[i]
```

Comment:
Looks like the indentation went in too many spaces. Still I appreciate that you are being conscious of your formatting.